mappings/com/mojang/blaze3d/framebuffer/Framebuffer.mapping

```diff
@@ -1,23 +1,28 @@
-CLASS net/minecraft/unmapped/C_xxwgvelc net/minecraft/client/gl/Framebuffer
+CLASS net/minecraft/unmapped/C_xxwgvelc com/mojang/blaze3d/framebuffer/Framebuffer
 	FIELD f_cfotkjmz textureWidth I
 	FIELD f_diuvorcl colorAttachment I
 	FIELD f_dzqtamlt useDepthAttachment Z
 	FIELD f_ecykjasv clearColor [F
 	FIELD f_haspsufq viewportWidth I
+	FIELD f_mpmtantv RED_CHANNEL I
 	FIELD f_mypruwdu depthAttachment I
-	FIELD f_sjyaqcdd texFilter I
+	FIELD f_olvnzwuo BLUE_CHANNEL I
+	FIELD f_sjyaqcdd textureFilter I
 	FIELD f_sxuwhuje textureHeight I
 	FIELD f_taldsayd viewportHeight I
-	FIELD f_wvqbkuit fbo I
+	FIELD f_ttgxaapu GREEN_CHANNEL I
+	FIELD f_wvqbkuit framebufferId I
+	FIELD f_yrdbqayj ALPHA_CHANNEL I
 	METHOD <init> (Z)V
 		ARG 1 useDepth
-	METHOD m_bzotkqxy setTexFilter (I)V
+	METHOD m_bzotkqxy setTextureFilter (I)V
+		ARG 1 filter
 	METHOD m_fddnofnq getDepthAttachment ()I
 	METHOD m_gcqqhxse drawInternal (IIZ)V
 		ARG 1 width
 		ARG 2 height
 		ARG 3 disableBlend
-	METHOD m_grzekhsz initFbo (IIZ)V
+	METHOD m_grzekhsz createBuffers (IIZ)V
```

Comment:
maybe just `init` or `create`? the only buffers being created in the method is the framebuffer, and the rest is texture setup.